l10n/configs/vendor.toml

```diff
@@ -0,0 +1,47 @@
+# All content handled by the vendor or Mozilla staff should
+# be covered by the [[paths]] in this file, with the relevant locales.
+
+basepath = ".."
+
+locales = [
+    "cs",
```

Comment:
The locale code for Czech is `cs`, `CZ` is the region code. This deviates from the spec, so calling that out.